src/vm/util.cpp

```diff
@@ -1388,7 +1388,7 @@ BOOL ReportEventCLR(
     GCX_PREEMP();
 
     SString buff;
-    buff.Printf(W(".NET Runtime version %s - %s"), VER_FILEVERSION_STR_L, message->GetUnicode());
+    buff.Printf(W(".NET Runtime version %s - %s"), VER_FILEVERSION_STR_L, message ? message->GetUnicode() : NULL);
```

Comment:
This method does not seem to be called from anywhere. It would be better to delete it.

Reply:
Removed `ReportEventCLR` from both cpp and hpp.